packages/components/psammead-navigation/src/index.test.jsx

```diff
@@ -2,34 +2,62 @@ import React from 'react';
 import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
 import { render } from '@testing-library/react';
 import { latin } from '@bbc/gel-foundations/scripts';
+import {
+  CanonicalSwipeableNavigation,
+  AmpSwipeableNavigation,
+} from './SwipeableNavigation';
 import Navigation, { NavigationUl, NavigationLi } from './index';
 import igboNavData from '../testHelpers/igbo';
 
+const NavigationExample = (
+  <Navigation script={latin} skipLinkText="Wụga n’ọdịnaya" service="news">
+    <NavigationUl>
+      {igboNavData.map((item, index) => {
+        const { title, url } = item;
+        const active = index === 0;
+
+        return (
+          <NavigationLi
+            key={title}
+            url={url}
+            script={latin}
+            active={active}
+            currentPageText="Current page"
+            service="news"
+            data-navigation="test_navigation"
+          >
+            {title}
+          </NavigationLi>
+        );
+      })}
+    </NavigationUl>
+  </Navigation>
+);
+
 describe('Navigation', () => {
+  shouldMatchSnapshot('should render correctly', NavigationExample);
+});
+
+describe('Swipeable Navigation', () => {
+  window.matchMedia = jest.fn().mockImplementation(query => {
+    return {
+      matches: false,
```

Comment:
```suggestion
      matches: true,
```
For the swipeable  navigation should the `matches` evaluate to true?